apps/api/src/app/benchmark/benchmark.controller.ts

```diff
@@ -94,4 +94,41 @@ export class BenchmarkController {
       );
     }
   }
+
+  @Post('remove')
+  @UseGuards(AuthGuard('jwt'))
+  public async deleteBenchmark(@Body() { dataSource, symbol }: UniqueAsset) {
```

Comment:
`@Param('dataSource') dataSource: DataSource, @Param('symbol') symbol: string`

Reply:
Resolved in latest commit.